circle.yml

```diff
@@ -6,7 +6,9 @@ machine:
 
 dependencies:
   override:
+    - brew install swiftlint
     - bundle install
+    - rake danger
```

Comment:
You might find life easier by moving this last given the "finding the PR issues"